.github/workflows/ci.yml

```
@@ -1,91 +1,32 @@
 name: CI
 
 on:
-  push:
   pull_request:
-  schedule: # Trigger a job on master at midnight UTC every day
-    - cron: '0 0 * * *'
+  push:
+    branches:
+      - ros2
+      - master
+      - ci-dev 
```

```suggestion
      - ros2
```

This should only list ros2 for now.

when we merge ros2 into master, we'll replace ros2 here with master**